### ISSUE TITLE
pretrained model

### ISSUE THREAD

**Issue by Maryam483** [CellEight/Pytorch-Adaptive-Instance-Normalization#2]
The link to pretrained model is not working??? how can i download the model?
